### DISCUSSION TITLE
MAUI IdentityServer Samples?

### DISCUSSION THREAD

**Discussion by ElliottBrand** [dotnet/maui#1217] [Q&A]
Are there any samples out there that show how to integrate IdentityServer4 with MAUI?  I already have Blazor sites setup with IdentityServer, so this will be needed in order to build native Blazor apps for them.  I assumed I'd just add a few nuget packages and add some lines to the Startup.cs file in the main MAUI project, but I haven't had much luck so far.

Since most of my app is Blazor running in the BlazorWebView, that also complicates it when seeking examples that show how to wire it all up.

**Comment by ricardoromaobr** [comment]
I'm interest in this too. I see how to authenticate with the Maui only: [Authenticate with identityserver](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiu_r6W0Z_8AhWUqJUCHUm9D9IQwqsBegQICBAF&url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DPtYiz45xIJ8&usg=AOvVaw0p5MQuZwc66dZ05oilBo9Y), but with Blazor Maui I not found an example. 

I don't know with BlazorWebView will accept that same  settings that the Blazor only accept?
